tests/integration/test_cases/itest_metrics.yaml

```diff
@@ -1889,3 +1889,21 @@ integration_test:
         ds
     ) b
     ON {{ render_date_sub("a", "ds", 5, TimeGranularity.DAY) }} = b.ds
+---
```

Comment:
To add:
+---
+integration_test:
+  name: query_with_metric_in_where_filter
+  description: Tests a query with a metric in the where filter
+  model: SIMPLE_MODEL
+  metrics: ["listings"]
+  where_filter: "{{ render_metric_template('bookings', ['listing']) }} > 2"
+  check_query: |
+    SELECT 1

and remove the group by one